tests/acceptance/object-inspector-test.js

```diff
@@ -924,8 +902,6 @@ module('Object Inspector', function (hooks) {
   });
 
   test('Date fields are editable', async function (assert) {
```

Comment:
not sure why this test is failing now.
I see that we have 2 `respondWith('objectInspector:saveProperty')`, but we are only doing 1 save.
So I think you can just remove the first `respondWith('objectInspector:saveProperty')`

Reply:
@patricklx hmm, yes that does seem odd, however, the boolean tests are failing too, and that one just has one `respondWith`. I will try removing what you said though and see if anything changes.